bindings/kafka/kafka.go

```diff
@@ -89,9 +89,12 @@ func (b *Binding) Read(ctx context.Context, handler bindings.Handler) error {
 		return nil
 	}
 
-	ah := adaptHandler(handler)
+	handlerConfig := kafka.SubscriptionHandlerConfig{
+		IsBulkSubscribe: false,
```

Comment:
Nit: is this necessary, since `false` is the zero value? (although adding it for clarity may help)

Reply:
Yeah, this is by default, but just want to extra clear here, so wanted to set.